blockchain/blockchain-wallet-v4-frontend
# Resend SMS Codes

## Description
Adds a button to login form to resend SMS codes for 2FA enabled wallets. Also fixes an issue where 2FA login was broken.

## Change Type
Feature & Bug Fix

## Testing Steps
Create a wallet with SMS 2FA enabled.  Ensure you can login into wallet using initial SMS code as well as a resent SMS code.

## PR Creator Checklist

## PR Reviewer Checklist
